docs/pages/architecture/soverign-rollups.md

```diff
@@ -0,0 +1,53 @@
+# Sovereign Rollup Using Celestia and Dojo
+
```

Comment:
we should add an image here explaining this

Reply:
in anycase - can merge now and do later

Reply:
yup waiting on click to put it together